core/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -1186,7 +1188,7 @@ class QualificationSuite extends BaseTestSuite {
               // "SQL DF Duration" and "GPU Opportunity" cannot be larger than App Duration
               assert(numValue >= 0 && numValue <= appDur)
             } else {
-              assert(numValue > 0)
+              assert(numValue >= 0)
```

Comment:
nit: Should we modify this condition or include a separate `if` condition for `TOTAL_CORE_SEC` header similar to the others?

Reply:
updated this by rounding up total core sec, otherwise 800ms will result to 0s which seems confusing, thanks!